samples/boards/nordic/coresight_stm/Kconfig.sysbuild

```diff
@@ -4,6 +4,22 @@
 
 source "share/sysbuild/Kconfig"
 
+config CORESIGHT_STM_USE_PPR
+	bool "Use PPR"
+
+config CORESIGHT_STM_USE_FLPR
+	bool "Use FLPR"
+
 config RAD_CORE_BOARD
-string
+	string
 	default "nrf54h20dk/nrf54h20/cpurad" if $(BOARD) = "nrf54h20dk"
+
+config PPR_CORE_BOARD
+	string
+	default "nrf54h20dk/nrf54h20/cpuppr" if $(BOARD) = "nrf54h20dk"
+	depends on CORESIGHT_STM_USE_PPR
+
+config FLPR_CORE_BOARD
+	string
+	default "nrf54h20dk/nrf54h20/cpuflpr" if $(BOARD) = "nrf54h20dk"
+	depends on CORESIGHT_STM_USE_FLPR
```

Comment:
loops a bit hacky, why is this needed? samples like sysbuild/hello_world stopped hardcoding board names in Kconfig.sysbuild, please check. You can use sample.yaml to specify supported builds.

Reply:
I think this sample is a bit more complicated than sysbuild/hello_world. Besides app core there will be some number of "remotes".
On nrf54l20 we would like to heave all possible remotes - Radio, PPR, FLPR.
However, number of remotes can be different for other chips.
I believe, it would be good to have possibility to enable different number of remotes (APP+Radio or APP+Radio+PPR or APP+PPR or etc.) and select different target for each remote.

Reply:
See sample.yaml - there are three configurations. When run on nrf54h20:

1. sample.boards.nrf.coresight_stm.dict will run on APP + Radio + PPR + FLPR
2. sample.boards.nrf.coresight_stm will run on APP + Radio + PPR + FLPR
3. sample.boards.nrf.coresight_stm.local_uart will run on APP + Radio

Reply:
Reworked it to use single dir (`remote`) for rad,ppr and flpr. Removed hardcoded boards from `Kconfig.sysbuild`. @gmarull can you take a look?